state_machine/drivers/emulation/lib/pycubed.py

```diff
@@ -25,6 +25,16 @@ def duty_cycle(self, duty_cycle):
 class HardwareInitException(Exception):
     pass
 
+class rtc_sensor:
+    def __init__(self, dateInput):
+        self.datetime = dateInput
+
+    def updateTime(self, dateInput):
```

Comment:
We've generally been using underscores rather than case changes to separate words in functions and arguments. If you could use `date_input` and `update_time` that would be great.